rules.md

```diff
@@ -0,0 +1,18 @@
+# Substrate
+
+- 🔒️ed lines are reviewed by 2 team leads, one of them must be in @paritytech/locks-review and the other in @paritytech/polkadotleads 
```

Comment:
`and the other in @paritytech/polkadotleads `

It could still be that the second reviewer is also in locks-review. As locks-review is a subset of the other group.

Reply:
> Basically, the corner case here would be not to accept reviews as sufficient when there are two reviews from `@paritytech/polkadotleads` and none from `@paritytech/locks-review`

I have seen the other case as the corner case, but in the end it isn't that important as long as both cases are ensured :)